docs-src/references/dynamic-configuration.md

```diff
@@ -61,65 +61,47 @@ testGetMapPropertyKey:
-      value: "value-for-namespace1"
+      value: "4000"
     - constraints: {namespace: "namespace2"}
-      value: "value-for-namespace2"
+      value: "1000"
   ```
 
 - For keys that can be customized at a Task Queue level, you can specify Task Queue name and Task type in addition to Namespace.
-  To set values at a Task Queue level, use `taskqueueName` (String) with `taskType` (optional; supported values: `Workflow` and `Activity`), as shown in the following exmaple.
+  To set values at a Task Queue level, use `taskqueueName` (String) with `taskType` (optional; supported values: `Workflow` and `Activity`).
+
+  For example to override the default maximum queries per second made to the Persistence database from the Frontend Service, add the following to your dynamic configuration file.
 
   ```yaml
-  keyName:
-  - constraints: {namespace: "namespace1", taskQueueName: "tq"}  #Applies to both Workflow and Activity tasks on the "tq" Task Queue.
-    value: 44
-  - constraints: {namespace: "namespace1", taskQueueName: "other-tq", taskType: "Activity"} #Applies to Activity tasks on the "other_tq" Task Queue.
-    value: 33
-  - constraints: {namespace: "namespace2"}  #Applies to all task queues in "namespace2".
-    value: 22
+  frontend.persistenceMaxQPS:
+  - constraints: {namespace: "namespace1", taskQueueName: "tq"}  # Applies to the "tq" Task Queue for both Workflows and Activities.
+    value: 3500 # The default value for this key on the Frontend Service is 2400.
+  - constraints: {namespace: "namespace1", taskQueueName: "other-tq", taskType: "Activity"} # Applies to the "other_tq" Task Queue for Activities specifically.
+    value: 4000 
+  - constraints: {namespace: "namespace2"}  # Applies to all task queues in "namespace2".
+    value: 1500
   - constraints: {}  # Applies to all other task queues in "namespace1" and all other Namespaces.
-    value: 11
+    value: 3000
   ```
 
-Note that these values are not applied first to last; values set on specific constraints override default values defined for that configuration key.
-
-For example to override the default maximum queries per second made to the Persistence database from the Frontend Service, add the following to your dynamic configuration file.
-
-```yaml
-#...
-frontend.persistenceMaxQPS:
-  - value: 3000 # The default value for this key on the Frontend Service is 2000.
-    constraints: {}
-#...
-```
-
-You can also set the maximum queries that can be made from a Namespace on the Frontend with the `frontend.PersistenceNamespaceMaxQPS` key.
-
-```yaml
-#...
-frontend.PersistenceNamespaceMaxQPS:
-  - value: 3500 # The default value for this key on the Frontend Service is 2000.
-    constraints:
-      namespace: "your-namespace"
-#...
-```
+Note that the values set with most constraints take priority over values that are set with fewer constraints, regardless of the order in which they are set in the dynamic configuration key.
```

Comment:
this isn't technically correct but it's probably close enough for this purpose. (that's why I wrote "more constrained" in quotes... the exact logic is probably too complicated and boring to explain)

Reply:
Would it be better if I removed this point?